update-antora-spring-ui/action.yml

```diff
@@ -0,0 +1,88 @@
+name: Update Antora UI Spring
+description: A GitHub Action that detects if there is a new version of the Antora UI Spring artifact and creates a PR to update the antora-playbook.yml file to use the new version.
+
+inputs:
+  docs-branch:
```

Comment:
It is helpful to update the docs-build branch, but we should consider that we should also be able to send PRs for the currently active branches (e.g. [main](https://github.com/spring-projects/spring-security/blob/0a6e50d892b1280d7f668226158a0387b20ba830/docs/antora-playbook.yml#L33), [6.2.x](https://github.com/spring-projects/spring-security/blob/9dd0597338721de4177f6864b66104f3ee000d3c/docs/antora-playbook.yml#L33), etc)

Reply:
I've added to the README how to make it run for multiple branches:

```yml
jobs:
  update-antora-ui-spring:
    runs-on: ubuntu-latest
    name: Update Antora UI Spring
    strategy:
      matrix:
        branch: ['main', 'docs-build']
    steps:
      - uses: spring-io/spring-docs-actions/update-antora-spring-ui@{ACTION_VERSION}
        name: Update antora-playbook.yml
        with:
          docs-branch: ${{ matrix.branch }}
          token: ${{ secrets.GITHUB_TOKEN }}
```